bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -1012,6 +1012,12 @@ public void scanEntryLog(long entryLogId, EntryLogScanner scanner) throws IOExce
             while (true) {
                 // Check if we've finished reading the entry log file.
                 if (pos >= bc.size()) {
+                    LOG.warn("read position illegal, ledgersMap maybe error. position {} entrylog size {} entrylog {}",
+                            pos, bc.size(), entryLogId);
+                    break;
+                }
```

Comment:
`pos >= bc.size` used checking if we've finished reading the entry log file, why need add warn log? Does this mean the warn log is always there?